source/openapi-admin-v3.yaml

```diff
@@ -557,31 +557,48 @@ paths:
       tags:
         - services
       operationId: adminListServices
-      summary: List all services within an application.
+      summary: List all data sources and third-party services
       description: |-
-        List all [services](https://www.mongodb.com/docs/realm/services#std-label-services) within an
-        application.
+        List all [data sources](https://www.mongodb.com/docs/atlas/app-services/mongodb/) and [[Deprecated] third-party
```

Comment:
One nit throughout: the link says "[Deprecated] third-party services" but the page title is written as "Third-Party Services [Deprecated]". I prefer the latter and personally would update the wording of these links to match. Serious nit, tho